config/ortho_bored.yml

```diff
@@ -1,75 +1,77 @@
-    │  {{0}}  │ {{1}} ├───┤ {{15}} ├───┼───┤       ├───┼───┤ {{20}} ├───┤ {{10}} │  {{11}}  │
-    ├─────┼───┤ {{14}} ├───┤ {{16}} │ {{17}} │       │ {{18}} │ {{19}} ├───┤ {{21}} ├───┼─────┤
-    │  {{12}}  │ {{13}} ├───┤ {{27}} ├───┼───┤       ├───┼───┤ {{32}} ├───┤ {{22}} │  {{23}}  │
-    ├─────┼───┤ {{26}} ├───┤ {{28}} │ {{29}} │       │ {{30}} │ {{31}} ├───┤ {{33}} ├───┼─────┤
-    │  {{24}}  │ {{25}} ├───┤ {{39}} ├─━─┼───┤       ├───┼─━─┤ {{44}} ├───┤ {{34}} │  {{35}}  │
-    ├─────┼───┤ {{38}} ├───┤ {{40}} │ {{41}} │       │ {{42}} │ {{43}} ├───┤ {{45}} ├───┼─────┤
-    │  {{36}}  │ {{37}} ├───┘   └───┴───┘       └───┴───┘   └───┤ {{46}} │  {{47}}  │
-    ├─────┼───┼───┬───┬───┬───┐ ┌───┐ ┌───┬───┬───┬───┼───┼─────┤
-    │  {{48}}  │ {{49}} │ {{50}} │ {{51}} │ {{52}} │ {{53}} │ │ {{54}} │ │ {{55}} │ {{56}} │ {{57}} │ {{58}} │ {{59}} │  {{60}}  │
-    └─────┴───┴───┴───┴───┴───┘ └───┘ └───┴───┴───┴───┴───┴─────┘
+                ┌───┐                                          ┌───┐
+            ┌───┤ {{3}} ├───┬───┐                          ┌───┬───┤ {{10}} ├───┐
+    ┌───┬───┤ {{2}} ├───┤ {{4}} │ {{5}} ├───┐                  ┌───┤ {{8}} │ {{9}} ├───┤ {{11}} ├───┬───┐
+    │ {{0}} │ {{1}} ├───┤ {{17}} ├───┼───┤ {{6}} │                  │ {{7}} ├───┼───┤ {{24}} ├───┤ {{12}} │ {{13}} │
+    ├───┼───┤ {{16}} ├───┤ {{18}} │ {{19}} ├───┤                  ├───┤ {{22}} │ {{23}} ├───┤ {{25}} ├───┼───┤
+    │ {{14}} │ {{15}} ├───┤ {{31}} ├───┼───┤ {{20}} │                  │ {{21}} ├───┼───┤ {{38}} ├───┤ {{26}} │ {{27}} │
+    ├───┼───┤ {{30}} ├───┤ {{32}} │ {{33}} ├───┤                  ├───┤ {{36}} │ {{37}} ├───┤ {{39}} ├───┼───┤
+    │ {{28}} │ {{29}} ├───┤ {{45}} ├─━─┼───┤ {{34}} │                  │ {{35}} ├───┼─━─┤ {{50}} ├───┤ {{40}} │ {{41}} │
+    ├───┼───┤ {{44}} ├───┤ {{46}} │ {{47}} ├───┘                  └───┤ {{48}} │ {{49}} ├───┤ {{51}} ├───┼───┤
+    │ {{42}} │ {{43}} ├───┤ {{57}} ├───┼───┘┌───────┐        ┌───────┐└───┼───┤ {{68}} ├───┤ {{52}} │ {{53}} │
+    ├───┼───┤ {{56}} ├───┤ {{58}} │    │   {{62}}   │        │   {{63}}   │    │ {{67}} ├───┤ {{69}} ├───┼───┤
+    │ {{54}} │ {{55}} ├───┘   └───┘    ├───┬───┼───┐┌───┼───┬───┤    └───┘   └───┤ {{70}} │ {{71}} │
+    └───┴───┘                │   │   │   ││   │   │   │                └───┴───┘
+                             │ {{59}} │ {{60}} │ {{61}} ││ {{64}} │ {{65}} │ {{66}} │   
+                             └───┴───┴───┘└───┴───┴───┘  
 
 
   plot_template_short: |2
-    {{0}}{{1}}{{2}}{{3}}{{4}}{{5}} {{6}}{{7}}{{8}}{{9}}{{10}}{{11}}
-    ⇘{{12}}{{13}}{{14}}{{15}}{{16}} {{17}}{{18}}{{19}}{{20}}{{21}}⇘
-     {{22}}{{23}}{{24}}{{25}}{{26}} {{27}}{{28}}{{29}}{{30}}{{31}}
+    {{12}}{{13}}{{14}}{{15}}{{16}}{{17}} {{18}}{{19}}{{20}}{{21}}{{22}}{{23}}
+    {{24}}{{25}}{{26}}{{27}}{{28}}{{29}} {{30}}{{31}}{{32}}{{33}}{{34}}{{35}}
+    {{36}}{{37}}{{38}}{{39}}{{40}}{{41}} {{42}}{{43}}{{44}}{{45}}{{46}}{{47}}
```

Comment:
The "plot_template_short" only covers non-fixed keys and iterates them from 0 onward. This means, the numbers in the template should start with 0 and (if the total number of non-fixed keys remains identical to the standard keyboard) end with 31.

Reply:
Done.